src/utils/read-fiddle.ts

```diff
@@ -0,0 +1,62 @@
+import { fancyImport } from './import';
+import { EditorValues } from '../interfaces';
+import {
+  INDEX_HTML_NAME,
+  MAIN_JS_NAME,
+  PRELOAD_JS_NAME,
+  RENDERER_JS_NAME,
+  STYLES_CSS_NAME,
+} from '../shared-constants';
+
+import * as fsType from 'fs-extra';
+import * as path from 'path';
+
+/**
+ * Safely attempts to read a file.
+ * Don't crash the app if reading fails.
+ *
+ * @param {string} filePath
```

Comment:
Second param missing from JSDoc

Reply:
Fixed in c38dbe47ff4854f9d08bc7a8a35c5e76915f15bf